xml/storage_multipath.xml

```diff
@@ -4643,6 +4643,32 @@ Give root password for maintenance
      </para>
     </listitem>
    </itemizedlist>
+   <important>
+    <para>
+     At this stage, you are working in a <command>dracut</command> emergency
+     shell inside initrd. All modification resulting from the following
+     procedures will be lost at the next reboot. To make them persistent, you
+     need to either:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       <command>chroot</command> to the system root <filename>/</filename> file
```

Comment:
This does not say how to mount the / filesystem or where it is typically mounted by dracut.
In the emergency environment / is the ramdisk so it needs clarification that the system / is elsewhere.

Reply:
> This does not say how to mount the / filesystem or where it is typically mounted by dracut.
> In the emergency environment / is the ramdisk so it needs clarification that the system / is elsewhere.

Jiri Slaby suggested to not `chroot` at all as accessing config files is possible right after mounting. can you agree with that?

Reply:
that depends on what you want to do. for editing a file it might suffice to access is from the dracut shell, for running a command that makes use of the changed file or installing a package you need to chroot. also ramdisk typically does not have an editor

Reply:
I disagree with Jiri's suggestion. In my experience chrooting is necessary in most  cases. It's definitely necessary if the initrd needs to be rebuilt, and I can't think of many scenarios where it would make fixing the problem harder. 
True, if only `/etc/fstab` needs to be edited, it's not necessary. But how do you explain customers to differentiate between cases where **chroot** is necessary, and where it's not?